pages/bookings/index.tsx

```diff
@@ -4,37 +4,240 @@ import { BanIcon, CalendarIcon, CheckIcon, ClockIcon, XIcon } from "@heroicons/r
 import { DotsHorizontalIcon } from "@heroicons/react/solid";
 import { BookingStatus } from "@prisma/client";
 import dayjs from "dayjs";
-import { useRouter } from "next/router";
 import { Fragment } from "react";
+import { useMutation } from "react-query";
 
 import classNames from "@lib/classNames";
-import { trpc } from "@lib/trpc";
+import { HttpError } from "@lib/core/http/error";
+import { inferQueryOutput, trpc } from "@lib/trpc";
 
 import EmptyScreen from "@components/EmptyScreen";
 import Loader from "@components/Loader";
 import Shell from "@components/Shell";
 import { Alert } from "@components/ui/Alert";
 import { Button } from "@components/ui/Button";
 
+function BookingListItem(booking: inferQueryOutput<"viewer.bookings">[number]) {
```

Comment:
cc @baileypumfleet @emrysal @zomars 

good tip to know when you want to use the types from a tRPC-query's output